test/val/val_image_test.cpp

```diff
@@ -1571,10 +1544,10 @@ TEST_F(ValidateImage, SampleImplicitLodConstOffsetCubeDim) {
 
 TEST_F(ValidateImage, SampleImplicitLodConstOffsetWrongType) {
   const std::string body = R"(
-%img = OpLoad %type_image_f32_3d_0111 %uniform_image_f32_3d_0111
+%img = OpLoad %type_image_f32_2d_0001 %uniform_image_f32_2d_0001
```

Comment:
This is the first of many tests where a multisampled image was used only incidentally.
In each case I changed it to an ordinary sampled image.   The test file only had 2d, so the coordinates and offsets changed arity.